src/Firebase/firebase-integration.ts

```diff
@@ -35,7 +35,7 @@ export class FirebaseIntegration extends BaseFirebaseIntegration {
     }
 
     public sendUserClickedOnPlayEvent(): void {
-        this.customEvents('user_clicked', { click: 'Click' });
+        this.customEvents('tapped_start', { click: 'Click' });
```

Comment:
Let's have a quick huddle on this tomorrow, Amit, as I have some questions and refinements to go through 🙌🏽

Reply:
Sure George, sounds like a good plan to me🙌